nservicebus/dependency-injection/container-deprecation-notice.include.md

```diff
@@ -1 +1 @@
-WARN: Starting with NServiceBus version 8, the dependency injection container adapter packages are no longer required. NServiceBus directly supports the `Microsoft.Extensions.DependencyInjection.Abstractions` model and third party containers can be integrated using the [externally managed container mode](/nservicebus/dependency-injection/#externally-managed-mode). Visit the [dependency injection upgrade guide](/nservicebus/upgrades/7to8/dependency-injection.md) for further information.
\ No newline at end of file
+WARN: Starting with NServiceBus version 8, the dependency injection container adapter packages are no longer required. NServiceBus directly supports the `Microsoft.Extensions.DependencyInjection` model and third party containers can be integrated using the [externally managed container mode](/nservicebus/dependency-injection/#externally-managed-mode). Visit the [dependency injection upgrade guide](/nservicebus/upgrades/7to8/dependency-injection.md) for further information.
```

Comment:
I'm not sure this should change? The externally managed mode API references `IServiceCollection`, which is defined in `Microsoft.Extensions.DependencyInjection.Abstractions`.